docs/Design/Interfaces/Interface Methods.md

```diff
@@ -0,0 +1,116 @@
+# Interface Methods
+
+## Feature Overview
+
+Method declarations within interfaces, as well as variable declarations within interfaces.

```

Comment:
Can you explain a little more? Something related to saying that interfaces are allowed to etc. etc.?